Git assignment submission

bnusir opened the pull request:
I have learned the following:
1. how to create a new public or private repo
2. how to create a branch/fork
3. practice creating and editing a file
4. how to commit
5. how to add to staging
6. how to push changes
7. how to compare versions
8. practice creating a pull request
9. writing good comments

bnusir committed:
Update README.md

still working on the home work

bnusir committed:
Update README.md

bnusir committed:
I have answered the remaining questions

SidraBushra commented:
Assignment is incomplete, you did not answered all questions, and did not provide proper details of the assignment.

bnusir committed:
I have answered the d question

The grader asked me to answer all questions, so I checked and found I did not answer question d, in this update I answered question d.

bnusir committed:
more updates to question d

bnusir committed:
Update README.md

bnusir committed:
Update README.md

bnusir committed:
Update README.md

bnusir committed:
Update README.md

bnusir committed:
Update README.md

bnusir committed:
Update README.md

bnusir committed:
Update README.md

bnusir committed:
Update README.md

bnusir committed:
Updated the solutions for assignment resubmission

I have updated the solutions as follows:
1. I have added lines between the question and the answer
2. I have added 2 lines between the answer and the next question
3. I have answered question D which I missed in the first submission.
4. I have added the word Question in bold format before each question to make the text clearer.

bnusir committed:
Update README.md

bnusir committed:
Update README.md to resubmit the git assignment

I have updated the solutions as follows:
1. I have added lines between the question and the answer
2. I have added two lines between the answer and the next question
3. I have answered question D which I missed in the first submission.
4. I have added the word Question in bold format before each question to make the text clearer.
5. I have split answers for questions C & D on multiple lines for clarity.
6. I have format certain words in bold for readability.

bnusir commented:
I have updated my file to reflect the missing.  please advise how to
resubmit?
Thanks

On Fri, Aug 30, 2024 at 9:31 PM Sidra Bushra ***@***.***>
wrote:

> Assignment is incomplete, you did not answered all questions, and did not
> provide proper details of the assignment.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bnusir/git_assignment/pull/1#issuecomment-2322667895>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHEJSR2K2VCBSMIDLUYU3U3ZUEMHPAVCNFSM6AAAAABNBFNKPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMRSGY3DOOBZGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Bashar Nusir

SidraBushra commented:
Hi, you can commit changes by using git commit -m and then push the changes to the branch.

bnusir commented:
Thanks,
   can you check and confirm if you can see the updates.
Thanks a lot

On Sat, Aug 31, 2024 at 10:04 PM Sidra Bushra ***@***.***>
wrote:

> Hi, you can commit changes by using git commit -m and then push the
> changes to the branch.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bnusir/git_assignment/pull/1#issuecomment-2323116967>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHEJSR6TLRQ64X6E75FF4DDZUJY4DAVCNFSM6AAAAABNBFNKPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMRTGEYTMOJWG4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Bashar Nusir

SidraBushra commented:
I can see the updates, and for last question, we usually don't push the changes directly to the main branch.

bnusir commented:
Thanks for the feedback!

On Tue, Sep 3, 2024, 8:05 p.m. Sidra Bushra ***@***.***>
wrote:

> I can see the updates, and for last question, we usually don't push the
> changes directly to the main branch.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bnusir/git_assignment/pull/1#issuecomment-2327653524>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHEJSR2MDCRGDOK4RW5AMHTZUZFGHAVCNFSM6AAAAABNBFNKPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMRXGY2TGNJSGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
